ebpfsvc/rpc_api.cpp

```diff
@@ -51,3 +51,55 @@ ebpf_server_verify_and_load_program(
     return EBPF_OPERATION_NOT_SUPPORTED;
 #endif
 }
+
+ebpf_result_t
+ebpf_server_verify_and_authorize_native_image(
+    /* [string][in] */ char* image_path)
+{
+    HANDLE native_image_handle = INVALID_HANDLE_VALUE;
+    ebpf_result_t result = EBPF_SUCCESS;
+    bool impersonated = false;
+    if (RpcImpersonateClient(nullptr) != RPC_S_OK) {
+        result = EBPF_ACCESS_DENIED;
+        goto Exit;
+    }
+
+    impersonated = true;
+
+    native_image_handle = CreateFileA(
```

Comment:
I notice this is using an ANSI API instead of treating it as UTF-8.  Should it be UTF-8 (e.g., convert UTF8 to UTF16LE and then use CreateFileW)?